flask_dance/consumer/oauth2.py

```diff
@@ -205,12 +207,19 @@ def authorized(self):
         url = URLObject(request.url)
         if request.headers.get("X-Forwarded-Proto", "http") == "https":
             url = url.with_scheme("https")
-        token = self.session.fetch_token(
-            self.token_url,
-            authorization_response=url,
-            client_secret=self.client_secret,
-            **self.token_url_params
-        )
+        try:
+            token = self.session.fetch_token(
+                self.token_url,
+                authorization_response=url,
+                client_secret=self.client_secret,
+                **self.token_url_params
+            )
+        except MissingCodeError:
+            raise MissingCodeError(
```

Comment:
Can you attach the original exception to the new exception that you're raising? That way, it's possible for code to introspect the new exception and figure out what the original exception was.